.gitignore

```diff
@@ -11,17 +11,15 @@ tdrs-frontend/yarn-debug.log*
 tdrs-frontend/yarn-error.log*
 tdrs-frontend/vars-frontend.yml
 tdrs-frontend/pa11y-screenshots/*
-tdrs-frontend/.env*
-
 
 #BACKEND
 tdrs-backend/vars-backend.yml
 tdrs-backend/coverage.xml
 tdrs-backend/htmlcov/*
+tdrs-backend/.env
```

Comment:
This is the key 🔑 change! 😄

Reply:
The space before `.env` was actually causing this file to not be ignored by git before.

Reply:
Sorry, could you elaborate on this one? I don't see any whitespace in line 19

Reply:
This entry was previously on line 24, I moved it up with the others that start with `tdrs-backend` just for OCD purposes

Reply:
Ah. I see. Great catch.